openedx2zim/scraper.py

```diff
@@ -490,30 +487,33 @@ def download_from_youtube(self, url, fpath):
     def convert_video(self, src, dst):
         if (src.suffix[1:] != self.video_format) or self.low_quality:
             preset = VideoWebmLow() if self.video_format == "webm" else VideoMp4Low()
-            reencode(
+            return reencode(
                 src, dst, preset.to_ffmpeg_args(), delete_src=True, failsafe=False,
             )
 
-    def optimize_image(self, src, dst, resize=None):
-        if resize:
-            resize_image(src, width=resize, allow_upscaling=True)
-        if src.suffix[1:] == "jpeg":
+    def optimize_image(self, src, dst):
+        optimized = False
+        if src.suffix in [".jpeg", ".jpg"]:
             exec_cmd("jpegoptim --strip-all -m50 " + str(src), timeout=10)
-        elif src.suffix[1:] == "png":
+            optimized = True
```

Comment:
What about

```py
optimized = exec_cmd("jpegoptim --strip-all -m50 " + str(src), timeout=10) == 0
```

Instead of having two lines for each?

Reply:
Did that (except for PNG as that needed two steps)

Reply:
For PNG, do it on second step only.